docs/HowTo/BreakingChanges.md

```diff
@@ -0,0 +1,142 @@
+# Breaking Changes
+
+Compatibility is a very important goal of each .NET release.  In order to achieve this, various techniques are employed. As shown
+earlier, there is additive functionality with each version, so previous versions will still work.  However, when a change must be 
+made to previous functionality (for performance, security, or a bug) this can cause compatibility problems in already written code. 
+A distinction to understand with .NET compatibility is runtime vs retargeting issues.
+
+When developing an app, the .NET Framework version that the app is targeting is [specified either by specifying a target Framework 
+in Visual Studio](https://msdn.microsoft.com/en-us/library/bb398202%28v=vs.140%29.aspx), directly 
+[specifying the target Framework](https://msdn.microsoft.com/en-us/library/hh264221.aspx) in a project file, or applying a 
+[TargetFrameworkAttribute](https://msdn.microsoft.com/en-us/library/system.runtime.versioning.targetframeworkattribute%28v=vs.110%29.aspx) 
+to the source code.
+
+An app that is built targeting a particular version of the .NET Framework (4.0 or above) will run on that version of the .NET 
+Framework or on any newer in-place update version. When running on a newer version than what was targeted, the .NET Framework will 
+use quirked behavior to mimic the older targeted version. The app will run on the newer version of the Framework, but act as if it's
+running on the older version. Many of the breaking changes between versions of the .NET Framework are mitigated through this 
+quirking model.
+
+Runtime issues are when a new runtime is placed on a machine and the same binaries are run. If a binary was compiled for .NET 4.0,
+it will then run in .NET 4.0 compatibility mode on 4.5+. Many of the changes that affect 4.5 will not affect a binary compiled
+for 4.0. This is specific to the AppDomain and depends on the settings of the entry assembly. 
+
+Retargeting issues are those that arise when an assembly that was targeting 4.0 is now set to target 4.5. Now, it opts into many of
+the new features, with some breaking changes to old features. Again, this is dictated by the entry assembly, so the console app that
+uses the assembly, or the website that references the assembly.
+
+Sometimes, there are new behaviors that can be toggled on or off. This is often done by registry keys or configuration values, but
+depends on the specific change if those are available.
+
+This analysis is only applicable to assemblies that are designed to run on the .NET Framework 4.0+.  This is because the .NET 
+Framework is an in-place update that has compatibility modes for previous versions of the framework. This means that you can update
+to the newest version, currently 4.6, and still be able to target modes for 4.0, 4.5, 4.5.1, and 4.5.2.  This is done through a 

```

Comment:
Nitpick - I feel like some of the beginning of this paragraph is unnecessary since the doc has just gone over the difference between runtime and retargeting breaks.